src/renderer/components/duration/reactive-duration.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { observer } from "mobx-react";
+import { now } from "mobx-utils";
+import React from "react";
+import { formatDuration } from "../../utils";
+
+export interface ReactiveDurationProps {
+  timestamp: string;
+
+  /**
+   * Whether the display string should prefer length over precision
+   * @default true
+   */
+  compact?: boolean;
+}
+
+/**
+ * This function computes a resonable update
+ */
+function computeUpdateInterval(creationTimestampEpoch: number): number {
+  const seconds = Math.floor((Date.now() - creationTimestampEpoch) / 1000);
+  const minutes = Math.floor(seconds / 60);
+
+  if (minutes < 10) {
+    // Update every second
+    return 1000;
+  }
+
+  return 60 * 1000;
+}
+
+export const ReactiveDuration = observer(({ timestamp, compact = true }: ReactiveDurationProps) => {
+  const creationTimestamp = new Date(timestamp).getTime();
+
+  return (
+    <>
+      {formatDuration(now(computeUpdateInterval(creationTimestamp)) - creationTimestamp, compact)}
```

Comment:
IMHO we need to tweak `formatDuration` (or use other formatter) now that we update/compute the duration. Updating things like `2m3s` every second will be annoying. My suggestion is to drop seconds completely when over `1m` and then update duration only once per minute.

Reply:
Just realized that current `formatDuration` mimics kubectl output. Maybe it's ok to keep using that.

Reply:
I can tweak it if you like...

I could slow down the updates to every 5s if you think it is too fast, and thus distracting.